src/slices/widget.ts

```diff
@@ -4,23 +4,20 @@ import { WidgetConfig } from "types/widget";
 type State = WidgetConfig;
 
 const initialState: State = {
-  endowment: { id: 0, name: "Nonprofit name" },
-  isDescriptionTextShown: true,
-  advancedOptions: {
-    display: "expanded",
-    liquidSplitPct: 50,
-  },
+  endowment: { id: 0, name: "", hide_bg_tip: false },
+  hideDescription: false,
+  isSplitDisabled: false,
+  liquidSplitPct: 50,
 };
 
 const widget = createSlice({
   name: "widget",
   initialState,
   reducers: {
-    update: (_, { payload }: PayloadAction<State>) => payload,
-    reset: () => initialState,
+    updateWidgetConfig: (_, { payload }: PayloadAction<State>) => payload,
```

Comment:
@misicnenad @ap-justin I would lean towards the use of more standard form like `update`/`reset` over the longer `updateWidgetConfig` change.

Reply:
@SovereignAndrey @ap-justin quoting the message of the [commit that introduced the change](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2802/commits/6c9d5f80a5609e7561c3dbf995f2a8293f52425d):
> give unique names to widget.reducer actions (fix bug where resetting widget config would actually call auth.reset)

In other words, as all state reducers would get merged into one in *src/store/store.ts*, calling the `slices/widget > reset` action conflicted with `slices/auth > reset`, causing `state.user` to be set to null and the page to redirect to `/signin`. You can repro the bug here https://staging.better.giving.

Also, although less important, widget actions action were being renamed anyway just before being exported (see previous [line 25](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2802/files#diff-447fabfdac84c5293d0fc27c4f191b41d01f2b295d81f4d702bcb0492bc86338L25)):
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/5d1c8a3a-afe1-41ca-8e13-40ff71f4b33a)

Reply:
@misicnenad 
> In other words, as all state reducers would get merged into one in src/store/store.ts,

found the bug to be in`slices/auth` where the `slice` name is `widget` instead of `auth` - I believe this was my mistake.

Reply:
Ooh, so that was the reason for the bug. 
Updated widget slice actions to use simple names and fixed the bug you pinpointed, see https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2802/commits/1fe3b2aaf5ff14215f159649fb4e3c656b6aaebe.